### ISSUE TITLE
Fix internal error on bad class name

### ISSUE THREAD

**Issue by wsnyder** [verilator/verilator#5475]
The ClassRigh typo below results in an internal error:

```
class ClsRight;
   string m_s;
endclass

module t (/*AUTOARG*/);
   string s;
   initial begin
      // verilator lint_off PKGNODECL
      s = ClsRigh::m_s;  // Bad typo
   end
endmodule

```

See test_regress/t/t_class_ref_bad.v

**Comment by Furqan-Nadir** [comment]
Hey @wsnyder can you assign me this task? I think I can help with the issue.

**Comment by wsnyder** [comment]
Great, glad to get a pull on this, thanks for looking at it.

**Comment by wsnyder** [comment]
Fixed with (#5629)
